### PR TITLE
Incorporate latest lang changes

### PR DESCRIPTION
**Pull request by daneshk**
## Purpose
$subject

### CODE REVIEW

File: grpc-ballerina/tests/17_bidirectional_chat_client.bal [hunk 1]
```diff
@@ -57,8 +57,10 @@ public isolated function testBidiStreamingInChatClient() returns Error? {
     }
 
     var responseMsg = ep->receive();
-    if (responseMsg is error) {
+    if responseMsg is error {
         test:assertFail(msg = responseMsg.message());
+    } else if responseMsg is () {
+        test:assertFail(msg = "stream can't be without a response message");
```
Comment (BuddhiWathsala) [inline grpc-ballerina/tests/17_bidirectional_chat_client.bal:63]: Isn't this the end of the stream?

**Review by BuddhiWathsala — CHANGES_REQUESTED** [review]

File: grpc-ballerina/tests/17_bidirectional_chat_client.bal (hunk 1 above)
Comment (daneshk) [reply grpc-ballerina/tests/17_bidirectional_chat_client.bal:63]: yes, but here we can expect a response message.

**Review by BuddhiWathsala — APPROVED** [review]

File: grpc-ballerina/tests/17_bidirectional_chat_client.bal (hunk 1 above)
Comment (daneshk) [reply grpc-ballerina/tests/17_bidirectional_chat_client.bal:63]: the error message is incomplete. I will fix it

Comment (daneshk) [inline grpc-ballerina/tests/17_bidirectional_chat_client.bal:63]:
```suggestion
        test:assertFail(msg = "stream can't be closed without a response message");
```

**Comment by BuddhiWathsala** [comment]
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/1209

**Pull request merged**